distributions/otelcol-k8s/manifest.yaml

```diff
@@ -0,0 +1,71 @@
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/k8s
+  name: otelcol-k8s
+  description: OpenTelemetry Collector for Kubernetes
+  version: 0.96.0
+  output_path: ./_build
+  otelcol_version: 0.96.0
+
+extensions:
```

Comment:
Would be good to see the jaeger remote sampling ext. here. wdyt?

Reply:
Since we're including `jaegerreceiver` I think you're right that we should include this extension.